src/System.Text.Json/tests/Utf8JsonReaderTests.MultiSegment.cs

```diff
@@ -7,6 +7,7 @@
 using System.Diagnostics;
 using System.IO;
 using Xunit;
+using System.Text.RegularExpressions;
```

Comment:
Is this using directive still needed? If not, please remove.

Reply:
Done!